Assets/Scripts/Tank/TankController.cs

```diff
@@ -7,19 +7,36 @@ namespace TankBattle.Tank
 {
     public class TankController
     {
+        private bool isBotTank = false;
+        private bool isLoaded;
```

Comment:
you can keep all public variable at top, then private so they are logically grouped together.